include/robot_localization/navsat_conversions.hpp

```diff
@@ -48,39 +48,39 @@
      @author Chuck Gantz- chuck.gantz@globalstar.com
  */
 
-#include <cmath>
-#include <string>
-
 #include <stdio.h>
 #include <stdlib.h>
 
+#include <cmath>
+#include <string>
+
 namespace robot_localization
 {
 namespace navsat_conversions
 {
 
-const double RADIANS_PER_DEGREE = M_PI/180.0;
-const double DEGREES_PER_RADIAN = 180.0/M_PI;
+const double RADIANS_PER_DEGREE = M_PI / 180.0;
+const double DEGREES_PER_RADIAN = 180.0 / M_PI;
 
 // Grid granularity for rounding UTM coordinates to generate MapXY.
-const double grid_size = 100000.0;    // 100 km grid
+const double grid_size = 100000.0;  // 100 km grid
 
 // WGS84 Parameters
-#define WGS84_A   6378137.0   // major axis
-#define WGS84_B   6356752.31424518  // minor axis
-#define WGS84_F   0.0033528107    // ellipsoid flattening
-#define WGS84_E   0.0818191908    // first eccentricity
-#define WGS84_EP  0.0820944379    // second eccentricity
+#define WGS84_A 6378137.0        // major axis
```

Comment:
Can we remove the extra spaces after these lines?

Reply:
Please don't mind. Here I didn't understand, which extra spaces need to be remove.  You mean to remove line 74?